Readme.md

```diff
@@ -2,38 +2,47 @@
 
 This script is intended as a simple QC method for Hi-C libraries, based on reads in a BAM file aligned to some genome/assembly. 
 
+The most informative Hi-C reads are the ones that are long-distance contacts, or contacts between contigs of an assembly. This tool quantifies such contacts and makes plots of contact distance distributions. The most successful Hi-C libraries have many long-distance and among-contig contacts.
+
 Hi-C connectivity drops off in approximately a power-law with increasing linear sequence distance. Consequently, one expects Hi-C reads to follow a characteristic distribution, wherein there is a spike of many read pairs at distances close to zero, which drops off smoothly (in log space) with increasing distance. If there are odd spikes or discontinuities, there may be a problem either with the library or the assembly.
 
 ## Dependencies
 * python2.7 (mostly tested using 2.7.10)
 * numpy
 * pysam
 * matplotlib
+* pdfkit
+* markdown
+* wkhtmltopdf (needs to be installed manually)
 
-These are installed with pip, e.g.:
+For installation, run this statement in a terminal:
 
-`pip install pysam`
+`git clone https://github.com/phasegenomics/bam_to_mate_hist.git && cd bam_to_mate_hist && sudo pip install -r requirements.txt`
```

Comment:
Better to suggest pip install --user -r requirements.txt rather than sudo pip install...